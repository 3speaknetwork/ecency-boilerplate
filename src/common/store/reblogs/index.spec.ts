import reducer, { initialState, reloadAct } from "./index";

jest.mock("../../util/local-storage", () => ({
  get: () => {
    return [
      {
        account: "lorem",
        author: "ipsum",
        permlink: "dolor",
      },
      {
        account: "foo",
        author: "bar",
        permlink: "baz",
      },
    ];
  },
}));

let state = initialState;

it("1- default state", () => {
  expect(state).toMatchSnapshot();
});

it("2- reloadAct", () => {
  state = reducer(state, reloadAct());
  expect(state).toMatchSnapshot();
});
