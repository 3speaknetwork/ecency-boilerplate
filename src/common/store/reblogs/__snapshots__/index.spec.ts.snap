// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`1- default state 1`] = `Array []`;

exports[`2- reloadAct 1`] = `
Array [
  Object {
    "account": "lorem",
    "author": "ipsum",
    "permlink": "dolor",
  },
  Object {
    "account": "foo",
    "author": "bar",
    "permlink": "baz",
  },
]
`;
